pymc3/tuning/starting.py

```diff
@@ -171,3 +176,126 @@ def allinmodel(vars, model):
     notin = [v for v in vars if v not in model.vars]
     if notin:
         raise ValueError("Some variables not in the model: " + str(notin))
+
+
+
+class Monitor(object):
```

Comment:
Not sure what the best test for this would be.   It runs and `Monitor` code is executed (but nothing is displayed of course) when `live_disp=True` is set outside of the notebook.   A test could just be to call it, and the test fails if there is any exception?